changelog.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to this project will be documented in this file.
 
+## 4.1.10 - 29 Jan 2020
+- IDM-389 - Fix to pass correctly hashed state to cache drop function
```

Comment:
_correctly pass_ instead of _pass correctly_ ?

Reply:
No. The fix is to correctly hash the state and then pass it